src/core_init_atmosphere/mpas_init_atm_cases.F

```diff
@@ -4497,8 +4497,12 @@ subroutine init_atm_case_gfs(block, mesh, nCells, nEdges, nVertLevels, fg, state
 
       if (allocated(maskslab)) deallocate(maskslab)
 
-      ! Freeze really cold ocean
-      where (sst < 271.0 .and. landmask == 0) xice = 1.0
+      ! Freeze really cold ocean: setting xice to 1.0 for sea-surface temperatures colder than 271K
+      ! has proved to produce too much seaice at high latitudes. The line below has been commented
+      ! out and setting xice to 1.0 for sea-surface temperatures colder than 100K is now moved to
+      ! the subroutine physics_init_seaice (in module mpas_atmphys_initialize_real.F).
+      ! Laura D. Fowler (laura@ucar.edu) / 2022-03-25.
```

Comment:
@ldfowler58 The commented out line is fine. The reason for this change is that in modern met dataset, fractional seaice, which is stored in xice, is typically provided. If one uses a temperature value to reset xice, it will potentially make fractional seaice to solid seaice for a mesh point. The 271 K seaice_threshold value is used in WRF to set seaice only if fractional seaice is not provided. This is the reason the default value for seaice_threshold is set to 100 K.

Reply:
Many thanks for the clarification. Makes sense to me.

Reply:
Should we update the PR description and code comments to incorporate the explanation of the 100K threshold from @weiwangncar ?

Reply:
@mgduda I agree with that. The comment inside the code can be made clearer too.

Reply:
As suggested, I included Wei's explantation to the description of the PR. Does this work for you?

Reply:
@ldfowler58 For this comment, it could say: " Freeze really cold ocean: setting xice to 1.0 for sea-surface temperatures colder than 271K has proved to produce too much seaice at high latitudes. The line below has been commented out.  Determination of using actual seaice fraction input or setting xice to 1.0 for sea-surface temperatures colder than a pre-defined value in the case where fractional seaice is not provided in the input is now moved to the subroutine physics_init_seaice (in module mpas_atmphys_initialize_real.F)."

Reply:
It looks like despite of what I write, this unfortunately does not fit your requirements. To avoid further back and forth, can one of you take over the writing as you wish? This would speed up the process.

Reply:
@ldfowler58 I'd suggest that we remove the comment as well as the line altogether, and just describe the change in the PR message.

Reply:
@ldfowler58 If it's alright with you, I could push some changes to the branch to update the code comment, commit message, and PR description. I like @weiwangncar's suggestion to make the code comment minimal, and to include in the commit message (and PR description) most of the explanation for why we're changing the code. But I can go with whatever you'd prefer.

Reply:
OK with me. I would really appreciate it. As I wrote above, this would speed up resolving this PR.

Reply:
I've just removed the commented-out code and comment altogether. The commit message now contains an explanation for the code change here.